From d4a37d7e99588685a81d75699b3bbd85621fbaea Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Tue, 21 May 2024 12:05:26 +0100
Subject: [PATCH 1098/1135] overlays: Add sdio -> sdio-pi5 mapping

There is already an sdio-pi5 overlays, but the automatic mapping was
missing.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/overlay_map.dts | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm/boot/dts/overlays/overlay_map.dts
+++ b/arch/arm/boot/dts/overlays/overlay_map.dts
@@ -267,6 +267,7 @@
 	sdio {
 		bcm2835;
 		bcm2711;
+		bcm2712 = "sdio-pi5";
 	};
 
 	sdio-1bit {
